src/plugins/drawing/model/drawing-content.ts

```diff
@@ -235,7 +270,7 @@ export const DrawingContentModel = TileContentModel
         setVectorType(vectorType: VectorType, ids: string[]) {
           self.vectorType = vectorType;
           forEachObjectId(ids, object => {
-            if (isVectorObject(object)) {
+            if (isVectorObject(object) || isGroupObject(object)) {
```

Comment:
Would it make more sense to just check to see if the object has `setEndShapes`, like you're doing frequently in `GroupObject` actions?

Reply:
It might make sense to change these to always directly check that the method exists.  Do you have any sense if the whole `getMembers(member).actions...` thing is efficient or better avoided?

Reply:
Alternatively, how about:
```
if ('setEndShapes' in object && typeof object.setEndShapes === 'function') {
    object.setEndShapes(...endShapesForVectorType(vectorType));
}
```